### PR TITLE
Update readme with build instructions

### PR DESCRIPTION
**Pull request by DrPeterVanNostrand**
Added to README:
1. Build requirements and instructions for installing the Protocol Buffer compiler.
2. Build instructions.

### CODE REVIEW

**Comment by DrPeterVanNostrand** [comment]
Awesome. That's good to know. Thanks

**Comment by afck** [comment]
We should mention that it requires protobuf 3, though, which is available as of [Debian 9 (Stretch)](https://packages.debian.org/search?keywords=protobuf-compiler&searchon=names&suite=all&section=all&sourceid=mozilla-search) and [Ubuntu 17.10 (Artful Aardvark)](https://packages.ubuntu.com/search?keywords=protobuf-compiler&searchon=names&suite=all&section=all).

And I'd like to note in the readme that this is work in progress for now. But maybe I'll add a list of the algorithms (Honey Badger itself, ABA, ACS, RBC, threshold signatures, threshold encryption, common coin, …) later anyway, with checkboxes to track the progress.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+somewhere in your `$PATH`. If you have not installed `protoc`, you can
+download the binary and move it to `/usr/local/bin/protoc` using the 
+following:
+
+*Note* As of writing this, the latest stable release of `protoc` is
+v3.5.1. You can check out what is the latest compiler version is
+[here](https://github.com/google/protobuf/releases), and update the
+following cURL url and zip-file name accordingly.
+
+## Installing `protoc` on Debain/Ubuntu
```
Comment (afck) [inline README.md]: Typo: "Debian"

Comment (DrPeterVanNostrand) [reply README.md]: good catch

**Comment by DrPeterVanNostrand** [comment]
@vkomenda  @afck I updated the readme with `protoc` v3 install instructions for Debian 9 and Ubuntu 17.

@afck Yea, the checkboxes would be cool!

**Pull request merged**